resources/extruders/atomPlus_extruder_0.def.json

```
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "Atom Plus",
```

```suggestion
        "machine": "atomPlus",
```

Same as the other one